staging/src/k8s.io/apiextensions-apiserver/pkg/apis/roundtrip_test.go

```diff
@@ -59,11 +62,30 @@ func TestCompatibility(t *testing.T) {
 	// limit to types in apiextensions.k8s.io
 	filteredKinds := []schema.GroupVersionKind{}
 	for _, gvk := range opts.Kinds {
-		if gvk.Group == apiextensionv1.SchemeGroupVersion.Group {
+		if gvk.Group == apiextensionsv1.SchemeGroupVersion.Group {
 			filteredKinds = append(filteredKinds, gvk)
 		}
 	}
 	opts.Kinds = filteredKinds
 
 	opts.Run(t)
 }
+
+func TestRoundtripToUnstructured(t *testing.T) {
```

Comment:
maybe add a comment that the types tested here are ones that don't *have* an "internal" go type, so this is just testing versioned → serialized → versioned round-tripping